### ISSUE TITLE
HG-20 High Gain Antenna - Localization

### ISSUE THREAD

**Issue by TykoTek** [PorktoberRevolution/ReStocked#374]
The descriptive text mentions both "direct" and "relay". Since this is a relay antenna it might be confusing to mention "direct". Suggest rewording to leave "direct" out.

![image](https://user-images.githubusercontent.com/22890228/52671479-13b1b780-2ed0-11e9-800f-3e0d9e931149.png)

**Comment by drewcassidy** [comment]
this is directly from the HG-5 description

> A short range dual purpose communications antenna that can handle either direct communications or short range relays
